docs/core/preview3/tools/dotnet-add-package.md

```diff
@@ -0,0 +1,81 @@
+---
+title: dotnet-add package command | Microsoft Docs
+description: The dotnet-add package command provides a convenient option to add NuGet package reference to a project.
+keywords: dotnet-add , CLI, CLI command, .NET Core
+author: spboyer
+ms.author: mairaw
+ms.date: 02/16/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 88e0da69-a5ea-46cc-8b46-5493242b7af9
+---
+# dotnet-add package
+
+[!INCLUDE[preview-warning](../../../includes/warning.md)]
+
+## Name
+
+`dotnet-add package` - Add package reference to a project file.
+
+## Synopsis
+
+```
+dotnet add [<PROJECT>] package <PACKAGE_NAME> [-v|--version] [-f|--framework] [-s|--source] [--package-directory] [-h|--help]
+dotnet add package [-h|--help]
+```
+
+## Description
+
+The `dotnet add package` command provides a convenient option to add a NuGet package reference to a project.
```

Comment:
We need to add more here. One thing to add would be that there is a compatibility check that happens. We should also state that the restore is ran after adding a package so that the asset file is complete. We should also add what is added to a project (a sample csproj fragment).